src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ProviderHandler.java

```diff
@@ -93,27 +62,25 @@ public static void removeProvider(final IStandardRequestManager manager, final I
     @SuppressWarnings(Suppression.UNCHECKED)
     public static void removeProviderInternal(final IStandardRequestManager manager, final IToken<?> token)
     {
-        final IRequestResolverProvider provider = getProvider(manager, token);
-
-        LogHandler.log("Removing provider: " + provider);
+        LogHandler.log("Removing provider: " + token);
 
         //Get the resolvers that are being removed.
-        final ImmutableCollection<IToken<?>> assignedResolvers = getRegisteredResolvers(manager, token);
+        final Collection<IToken<?>> assignedResolvers = getRegisteredResolvers(manager, token);
         for (final IToken<?> resolverToken : assignedResolvers)
         {
             //Skip if the resolver has no requests assigned.
-            if (!manager.getResolverRequestMap().containsKey(resolverToken) || manager.getResolverRequestMap().get(resolverToken).isEmpty())
+            if (!manager.getRequestResolverRequestAssignmentDataStore().getAssignments().containsKey(resolverToken) || manager.getRequestResolverRequestAssignmentDataStore().getAssignments().get(resolverToken).isEmpty())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 220 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)